packages/components/src/List/Toolbar/Filter/Filter.snapshot.test.js

```diff
@@ -39,4 +39,27 @@ describe('Filter', () => {
 		// then
 		expect(wrapper).toMatchSnapshot();
 	});
+
+	it('should render filter input with given placeholder', () => {
+		// when
+		const defaultProps = {
+			docked: false,
+			placeholder: 'find something',
+		};
+		const wrapper = renderer.create(<Filter {...defaultProps}/>).toJSON();
+
+		// then
+		expect(wrapper).toMatchSnapshot();
+	});
+
+	it('should render filter input with defaut placeholder', () => {
```

Comment:
Typo : `defaut ` --> `default `
Don't forget to update the snapshots after this fix